editor/editor_inspector.cpp

```diff
@@ -786,7 +786,7 @@ void EditorProperty::gui_input(const Ref<InputEvent> &p_event) {
 		}
 	} else if (mb.is_valid() && mb->is_pressed() && mb->get_button_index() == MOUSE_BUTTON_RIGHT) {
 		_ensure_popup();
-		menu->set_position(get_screen_position() + get_local_mouse_position());
+		menu->set_position(get_screen_transform().xform(get_local_mouse_position()));
```

Comment:
I think this one is ok. If something already uses screen position then it can be left like that.

Also for other changes, I'd do
`get_global_position()` -> `get_screen_position()`
`get_global_transform().xform()` -> `get_screen_transform().xform()`
i.e. if something uses position, just fix the position and if it uses transform, fix transform.

Reply:
Thanks. Done.